Sources/Testing/ABI/EntryPoints/SwiftPMEntryPoint.swift

```diff
@@ -27,7 +27,7 @@ var EXIT_NO_TESTS_FOUND: CInt {
 #if SWT_TARGET_OS_APPLE || os(Linux) || os(WASI)
   EX_UNAVAILABLE
 #elseif os(Windows)
-  ERROR_NOT_FOUND
+  numericCast(ERROR_NOT_FOUND)
```

Comment:
```suggestion
  CInt(ERROR_NOT_FOUND)
```

We don't otherwise use `numericCast` and we don't care about trapping here. :)